### PR TITLE
Add exclude_ucrs option to pillow runner

### PR DESCRIPTION
**Pull request by sravfeyn**
## Product Description
case, form and ucr pillows take this option but the pillow runner doesn't support this. This PR adds that support.

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

**Comment by sravfeyn** [comment]
@proteusvacuum

**Review by snopoke — COMMENTED** [review]
Would this be confusing since it only applies to selected pillows and not all?

File: corehq/ex-submodules/pillowtop/management/commands/run_ptop.py [hunk 1]
```diff
@@ -85,6 +85,13 @@ def add_arguments(self, parser):
             default=False,
             help="Set if you want to Move migrations on to side process",
         )
+        parser.add_argument(
+            '--exclude-ucrs',
+            dest='exclude_ucrs',
+            default="",
+            type=str,
+            help="comma-separated UCR tables IDS to exclude. Applicable for form/case/ucr pillows",
```
Comment (dannyroberts) [inline corehq/ex-submodules/pillowtop/management/commands/run_ptop.py:93]: Nit
```suggestion
            help="comma-separated UCR tables IDs to exclude. Applicable for form/case/ucr pillows",
```

**Review by dannyroberts — COMMENTED** [review]
is there a particular ticket that prompted this, or was it just an inconsistency that you happened to come across?

**Comment by sravfeyn** [comment]
> is there a particular ticket that prompted this, or was it just an inconsistency that you happened to come across?

@dannyroberts  This was prompted by eCHIS (a 3rd party self-hoster) who was trying to use this option and it didn't work since it missed this. There is a corresponding commcare-cloud PR https://github.com/dimagi/commcare-cloud/pull/4900

**Comment by sravfeyn** [comment]
@snopoke I have added that this is only relevant for form/case/ucr pillows in the help text for the arg, so hopefully not as confusing. Also, if they did use that option, the pillow would raise exception when run.

**Comment by sravfeyn** [comment]
@snopoke Wondering if this is g2g

**Review by snopoke — APPROVED** [review]

**Pull request merged**